src/engine/strat_engine/ns.rs

```diff
@@ -27,6 +27,8 @@ pub fn unshare_namespace() -> StratisResult<()> {
     if is_in_root_namespace()? {
         unshare(CloneFlags::CLONE_NEWNS)?;
     }
+    // unshare() seems to be working fine even if called on a PID 1 process, so the assertion below is
```

Comment:
Plz remove comment here and put a pre-condition in a comment above the method.

Suggested text is: "Precondition: If running in a container, PID is not 1 OR the container does not share the host PID.".

Feel free to revise if the wording is incorrect.

Reply:
Revised but kept the original intent. Thank you.